### PR TITLE
[WIP] Fix set_current_group for user data

### PR DESCRIPTION
**Pull request by jvlcek**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1733567

Small change to correct set_current_group on a user data

### CODE REVIEW

**Comment by jvlcek** [comment]
@miq-bot assign @abellotti
Please review

**Comment by jvlcek** [comment]
@himdel Please review

File: api/reference/users.adoc [hunk 1]
```diff
@@ -148,7 +148,7 @@ POST /api/users/:id
 {
   "action" : "set_current_group",
   "resource" : {
```
Comment (himdel) [inline api/reference/users.adoc]: This bit is apparently wrong.

This should be:

```
{
  "action" : "set_current_group",
  "current_group": { "id": "12" },
}
```

(gitter :point_up: [July 26, 2019 8:19 AM](https://gitter.im/ManageIQ/api?at=5d3ab7a2f4c906673aef1b4e))

Comment (jvlcek) [reply api/reference/users.adoc]: @himdel Thank you. Good catch. I incorrectly tested with "action" : "edit"

Fixing now.

**Comment by jvlcek** [comment]
Setting to WIP as I work out the difference between:
  "action" : "set_current_group",
and
  "action" : "edit",

**Comment by jvlcek** [comment]
@miq-bot add_label wip

**Comment by abellotti** [comment]
There is nothing wrong with the example in the doc.

for groups, "description" attributes are unique for groups and can be used as identifier as per
https://github.com/ManageIQ/manageiq-api/blob/master/config/api.yml#L1442

so group references can be via href, id or description  (attribute name is really misleading in the DB, should have been name).

**Comment by abellotti** [comment]
You could update the example in the doc mentioning that current_group is a group reference and a group can be identified via id, href or description.

**Comment by jvlcek** [comment]
This PR is not necessary

The documented syntax does work

e.g:

request.body = ' 
{
  "action" : "set_current_group",
  "resource" : {
    "current_group" : { "description" : "EvmGroup-super_administrator" }
  }
}
'

The set_current_group action can only be performed by the authenticated user.

**Pull request closed without merging**